docs/source/tutorials/parsl-rp.ipynb

```diff
@@ -0,0 +1,342 @@
+{
```

Comment:
<p>"Now, we import Parsl Python module in our application and the RadicalPilotExecutor" -&gt; "Now, we import the Parsl and RP Python modules in our application, alongside the RadicalPilotExecutor from Parsl"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ExaWorks/SDK/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>